### PR TITLE
chore(logging): Delete and log more

### PR DESCRIPTION
**Pull request by AndesKrrrrrrrrrrr**
We're logging user/pass in some of these logs. Removing, when possible, and minor refactor for clearer logging (ensuring no `Object [object]` logs.

## Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Formatting passes locally with my changes
- [ ] I have rebased against main before asking for a review

### CODE REVIEW

File: apps/skilavottord/ws/src/app/modules/recyclingRequest/recyclingRequest.service.ts [hunk 1]
```diff
@@ -89,11 +89,9 @@ export class RecyclingRequestService {
         )
       }
     } catch (err) {
-      throw new Error(
-        `Failed on deregistered vehicle ${vehiclePermno.slice(
-          -3,
-        )} because: ${err}`,
-      )
+      delete err.data
```
Comment (birkirkristmunds) [inline apps/skilavottord/ws/src/app/modules/recyclingRequest/recyclingRequest.service.ts:92]: What data are we deleting here? 
There is no data object in the error object, but there is a data in error.config, error.config.data

image

Comment (AndesKrrrrrrrrrrr) [reply apps/skilavottord/ws/src/app/modules/recyclingRequest/recyclingRequest.service.ts:92]: In one of these error logs, we are logging user/pass as the HTTP `--data` section. Updated the PR description

Comment (busla) [reply apps/skilavottord/ws/src/app/modules/recyclingRequest/recyclingRequest.service.ts:92]: I think there's also a generic error handler wrapper somewhere in `libs/shared`

**Review by birkirkristmunds — APPROVED** [review]
LGTM

**Pull request merged**